src/CommunityVoices/App/Website/Presentation/Module/Form/Quote.xslt

```diff
@@ -59,6 +59,21 @@
                             </div>
                         </div>
                     </xsl:if>
+                    <xsl:if test="domain/repeatedQuote = 'true'">
```

Comment:
How about making this an else if using `choose` so that we don't have to have the logic in the View?

Reply:
Okay, I'll switch it to a choose. A little confused though how this would be used in an else statement -- just because errors don't appear (the above if statement) doesn't necessarily mean that repeatedQuote will be true, am I missing something?

Reply:
Sorry, totally stupid comment I made before. I see it's an else if instead of an else.